### ISSUE TITLE
property_switch_exhaustiveness on a derived class

### ISSUE THREAD

**Issue by tpikonen** [bilelmoussaoui/gobject-lint#134]
Geoclue has this `property_switch_exhaustiveness` error:

```
src/gclue-modem-manager.c:198:1
  gclue_modem_manager_set_property (GObject      *object,
                                    guint         prop_id,
                                    const GValue *value,
                                    GParamSpec   *pspec)
  ...
          GClueModem *modem = GCLUE_MODEM (object);
  ...
  >        switch (prop_id) {
          case PROP_TIME_THRESHOLD:
                  gclue_modem_set_time_threshold (modem,
                                                  g_value_get_uint (value));
  error: Property 'PROP_IS_3G_AVAILABLE' should be handled in gclue_modem_manager_set_property switch statement
  rule: property_switch_exhaustiveness
```

`is-3g-available` is defined in the `GClueModem` interface as readable, but not writable, so I think it should not be included in the params handled by the `gclue_modem_manager_set_property` method in the derived class. This looks like a bug in handling overridden properties in gobject-linter.

**Comment by bilelmoussaoui** [comment]
Are you aware of the style config in https://bilelmoussaoui.github.io/gobject-linter/#property_switch_exhaustiveness? If you set it to legacy, it shouldn't complain about this case.

**Comment by tpikonen** [comment]
> If you set it to legacy, it shouldn't complain about this case.

Style setting seems to have no effect in this case.

I had a brief look at the code, and the problem stems from `access` being `None` in this particular case, here: https://github.com/bilelmoussaoui/gobject-linter/blob/62e82bc879bd27d5dd305fecc477eec2413a8d77/src/rules/property_switch_exhaustiveness.rs#L252
